src/pages/settings/Profile/LoginField.js

```diff
@@ -63,6 +64,14 @@ class LoginField extends Component {
         }
     }
 
+    /**
+    * Zero bottom margin for no phone or unverified phone
+    * @returns {Object}
+    */
+    getLabelMargin() {
+        return this.props.login.partnerUserID && !this.props.login.validatedDate ? styles.mb0 : {};
```

Comment:
> `this.props.login.partnerUserID` is for individual LoginField so it can be empty if the user has not added secondary login(it will never be empty for primary login).

Aah good point, thanks for reminding me there :D